### PR TITLE
Updates volume docs to support new volumes

### PR DESCRIPTION
**Pull request by jonnilundy**

### CODE REVIEW

File: node/electric/src/pages/docs/deploy/configuring-deployments/index.md [hunk 1]
```diff
@@ -214,19 +214,50 @@ There are three ways to scale your application.
...
-
-**Note:** Once you delete your project, any files in the volumes will also be destroyed.
+For many applications, it is necessary to be able to write and access a persistent file system so that your important files can keep their state even after re-deployment or restarting your service. We make this possible with `volumes`. 
 
 ```application/json
 {
   "id": "app",
-  "volume": "/opt/app/data"
+  "volumes": {
+    "store": "/opt/app/data"
```
Comment (zenorocha) [inline node/electric/src/pages/docs/deploy/configuring-deployments/index.md]: `store` sounds like an official key that the user has to follow, maybe we can change it to `foo`.

File: node/electric/src/pages/docs/deploy/configuring-deployments/index.md [hunk 2]
```diff
@@ -214,19 +214,50 @@ There are three ways to scale your application.
...
+  }
+}
+```
+
+Each volume (or file system drive), is mounted with a unique `id` that can be accessed by any service in that project. For example, if you create service1 and deploy with a volume declared as `documents`, you can access that same volume with service2 by declaring the same volume `id`
```
Comment (zenorocha) [inline node/electric/src/pages/docs/deploy/configuring-deployments/index.md]: Missing last period

File: node/electric/src/pages/docs/deploy/configuring-deployments/index.md [hunk 3]
```diff
@@ -214,19 +214,50 @@ There are three ways to scale your application.
...
 }
 ```
 
+**service2**
+
+```application/json
+{
+  "id": "service2",
+  "volumes": {
+    "document": "/photos"
```
Comment (zenorocha) [inline node/electric/src/pages/docs/deploy/configuring-deployments/index.md]: `document` > `documents`

**Comment by jonnilundy** [comment]
Thanks, @zenorocha. Updated with all the changes.

**Comment by zenorocha** [comment]
Also include changes to all the `volume` mentions on tutorials, docs, etc: https://github.com/wedeploy/wedeploy.com/search?utf8=%E2%9C%93&q=volume&type=

**Pull request merged**